web/src/components/tasks/business-formation/contacts/Addresses.tsx

```diff
@@ -121,7 +121,11 @@ export const Addresses = <T extends FormationMember | FormationIncorporator>(
           props.addressData.map((it, index) => {
             return (
               <Fragment key={index}>
-                <tr className="margin-bottom-1" key={index} data-testid={`${props.fieldName}-${index}`}>
+                <tr
+                  className={`margin-bottom-1 ${index % 2 === 1 ? "bg-base-lightest" : ""}`}
```

Comment:
[sand] so it seems like we reuse this code many times `${index % 2 === 1 ? "bg-base-lightest" : ""}`

so we could probably make this a function `const zerbraOnOdd = (index) =>  index % 2 === 1 ? "bg-base-lightest" : ""`

then use it like so `className={`margin-bottom-1 ${zebraOnOdd(index)}`}`